go.mod

```diff
@@ -275,3 +275,5 @@ replace (
 )
 
 exclude github.com/apache/pulsar-client-go/oauth2 v0.0.0-20211108044248-fe3b7c4e445b
+
+replace github.com/milvus-io/milvus-proto/go-api/v2 => github.com/shaoting-huang/milvus-proto/go-api/v2 v2.0.0-20241119072529-d67ec495e559
```

Comment:
why use private warehouse here？

Reply:
just make sure milvus and proto are both well. Will remove it once proto is merged